styles/tailwind.css

```
@@ -1,19 +1,63 @@
-  color utility to any element that depends on these defaults.
-*/
 @layer base {
-  *,
-  ::after,
-  ::before,
-  ::backdrop,
-  ::file-selector-button {
-    border-color: var(--color-gray-200, currentColor);
+  body {
+    @apply bg-aura-bg text-aura-text antialiased selection:bg-aura-primary/30;
+    background-image: radial-gradient(circle at 50% -20%, #8b5cf615 0%, transparent 50%);
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)